file_utils.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""
+Utilities for dealing with files.
+"""
+
+__author__ = "tomkinsc@broadinstitute.org"
+__commands__ = []
+
+import argparse
+import logging
+
+import util.cmd
+import util.file
+
+log = logging.getLogger(__name__)
+
+
+# ==============================
+# ***  merge_tarballs   ***
+# ==============================
+
+def merge_tarballs(out_tarball, in_tarballs, threads=None, extract_to_disk_path=None, pipe_hint=None):
+    ''' Merges separate tarballs into one tarball
+        data can be piped in and/or out
+    '''
+    util.file.repack_tarballs(out_tarball, in_tarballs, threads=threads, extract_to_disk_path=extract_to_disk_path, pipe_hint=pipe_hint)
+    return 0
+def parser_merge_tarballs(parser=argparse.ArgumentParser()):
+    parser.add_argument(
+        'out_tarball', 
+        help='''output tarball (*.tar.gz|*.tar.lz4|*.tar.bz2|-) 
+        Note: if "-" is used, a gzip-compressed tarball 
+        will be written to stdout''')
```

Comment:
Add some comment about how output compression is inferred by the file extension.